### PR TITLE
sci-astronomy/stellarium: add 0.19.1

### PR DESCRIPTION
**Pull request by DarthGandalf**
Signed-off-by: Alexey Sokolov <sokolov@google.com>
Package-Manager: Portage-2.3.66, Repoman-2.3.11

### CODE REVIEW

**Comment by DarthGandalf** [comment]
ping?

File: sci-astronomy/stellarium/stellarium-0.19.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="debug gps media nls stars test"
+
+BDEPEND="
+	nls? ( dev-qt/linguist-tools:5 )
+"
+
```
Comment (a17r) [inline sci-astronomy/stellarium/stellarium-0.19.1.ebuild]: unnecessary empty line

File: sci-astronomy/stellarium/stellarium-0.19.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	gps? ( dev-qt/qtpositioning:5 )
+	media? ( dev-qt/qtmultimedia:5[widgets] )
+"
+DEPEND="${RDEPEND}
+	dev-qt/qtconcurrent:5
+	test? ( dev-qt/qttest:5 )
+"
+
+src_prepare() {
+	xdg_environment_reset
```
Comment (a17r) [inline sci-astronomy/stellarium/stellarium-0.19.1.ebuild]: this is already called by `cmake-utils.eclass` in `src_configure`-

**Comment by a17r** [comment]
Well, not quite.
```
 * QA Notice: .desktop files with MimeType= were found installed
 * but desktop mimeinfo cache has not been updated:
 *   /usr/share/applications/stellarium.desktop
 * Please make sure to call xdg_desktop_database_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
 * QA Notice: mime-info files were found installed but mime-info
 * cache has not been updated:
 *   /usr/share/mime/packages/stellarium.xml
 * Please make sure to call xdg_mimeinfo_database_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
```

`0.19.0` had been correctly inheriting `xdg` to run these phases for you but it was dropped in `0.19.1` without comment. Will add it back after merge.

**Comment by DarthGandalf** [comment]
Ah. I assumed xdg is needed there only for xdg_environment_reset which is gone now.

I'll put it back now.

**Pull request closed without merging**